_posts/2021-11-08-changelog-159.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #159"
+date: 2021-11-08 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#7798] Support [let else](https://rust-lang.github.io/rfcs/3137-let-else.html) syntax available under `#![feature(let_else)]` feature gate
+
+  {% include gif-img.html path="/assets/posts/changelog-159/let_else" w="550px" %}
+
+* [#7898] Provide completion of the [`rust-version`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field) field in the `[package]` table of the `Cargo.toml` manifest (by [@HTG-YT])
+
+  <img src="/assets/posts/changelog-159/rust_version.png" alt="rust_version completion" width="550px">
+
+* [#7914] Use new name resolution for auto-import and out-of-scope completion. It supports auto-import of macros and significantly improves performance
```

Comment:
I think it's a most important feature of the release so let's move it at the first position